Activities menu and HSF Seminars (#1582)
HSF/hsf.github.io: activity-coordination.md

with a different subset of the developer and physics community and thus the
exact pattern of activities that are organised can vary. It is delegated to the
conveners to decide the way to organise so that it best serves the needs in
HEP software and computing as required.

### Meetings

#### Activity Topical Events

That said, organising semi-regular events is very important to keep up momentum
in a topic and sometimes a series of related events might happen bunched
together more tightly, e.g., looking at one specific topic from the point of
view of several experiments.

HSF activity meetings in general should try to:

- be there to help encourage the exchange of ideas and techniques
- do not need to focus on "finished" items, but rather can be there
  to provide a community forum for unsolved problems
- should ensure there is plenty of time for discussion
- should take notes or minutes for important discussion points (attaching these
  to Indico is recommended)

#### HSF Seminars and Cross Interest Meetings

Another route for a group is that instead of group specific meetings, organise
contributors to the [HSF Seminar Series]({{ site.baseurl

> }}/meetings/hsf-seminars.html). This meeting has a regular cadence and would

Reviewer: ```suggestion
}}/meetings/hsf-seminars.html). These meetings have a regular cadence and would
```

Reviewer 2: This I leave up to the native speaker

Author: Native speaker says yes, it's better as a plural!

for bringing together people that might not normally talk or know each other.
They might also fit into a seminar session.

#### Recording Meetings and Minutes

In general, recording *is appropriate* for HSF meetings, which are there to
foster discussion and collaboration, so *we do recommend it*. There are a lot of
meetings and we have a worldwide community, so not everyone who is interested
can join a meeting and recordings can help people to catch-up.

However, you need to make it very clear with the speakers and to participants that:

- The meeting is being recorded and will be made public;
- And where the video will be posted ([YouTube]({{ site.baseurl }}/organization/youtube.html), [CDS](http://cds.cern.ch/record/2289430), etc.);
  - Note that it is strongly recommended not to add video files directly into Indico, instead post a link to CDS.

In any case, *please also make sure that meetings are minuted in a concise way*
(it’s not always the case that people have the time to watch the 60 minute
video and they need a shorter summary). Minutes can be posted directly onto
Indico (easy) or to the [HSF website]({{ site.baseurl }}/howto-website.html) (via a PR).

> ### HSF Affiliated Projects

Reviewer: This is related to what we are preparing specifically. For the moment, what do you think of adding a new paragraph saying "A page dedicated to HSF Affiliated Projects and Software is under preparation, and will be linked here in due time."?

Reviewer 2: Better we just finish and add then. Forward declarations on websites sometimes can end up being just dead wood. Which we don't want to have :-)

Reviewer: That is certainly true :).

Well, to be fair, I did my share, I'm just waiting for "final" comments and an OK to go for those docs to be shared within the Forum. Let's do just that at this coming Monday meeting.

Reviewer 2: Absolutely! :-)

We strongly encourage the use of common software in the HSF and opportunities
for common development and sharing should be seized when they exist. The HSF
exists as a [GitHub organisation](https://github.com/HSF) and is happy
to host projects to help them grow. This can include things like common
datasets used in development as well as actual code.

### Coordinator Responsibilities

> Our working group conveners should usually dedicate between 5% and 10% of their

Reviewer: ```suggestion
Our activity conveners should usually dedicate between 5% and 10% of their
```

> time to running the working group. More than 10% is great, but we recognise

Reviewer: ```suggestion
time to running the activity area. More than 10% is great, but we recognise
```

> that people are busy with many other tasks. Less than 5% and the working group

Reviewer: ```suggestion
that people are busy with many other tasks. Less than 5% and the activity area
```

can fall into an inactive state. Conveners should meet at least once a month to
plan. The HSF Steering Group is always happy to attend to give feedback
and suggestions.

> At least one of the working group conveners should attend the bi-weekly

Reviewer: ```suggestion
At least one of the activity conveners should attend the bi-weekly
```

[HSF Coordination Meetings](https://indico.cern.ch/category/7970/) and
give a short update on the group's recent and forthcoming activities.

When the HSF organises [workshops](https://indico.cern.ch/category/7971/)

> working group conveners can give input to the organising team. If there

Reviewer: ```suggestion
activity conveners can give input to the organising team. If there
```

> will be a session dedicated to the activities of any working group then

Reviewer: ```suggestion
will be a session dedicated to matters of relevance to any activity area then
```
